VirtoCommerce.InventoryModule.Web/module.manifest

```diff
@@ -36,7 +39,7 @@
                 <defaultValue>Export/Import inventory information</defaultValue>
                 <title>Export/Import description</title>
                 <description>Module description in platform export/import wizard</description>
-            </setting>
+            </setting>           
```

Comment:
Whitespaces at the end of the line are not allowed.

Automatically format documents when saving:
Install https://marketplace.visualstudio.com/items?itemName=VisualStudioPlatformTeam.PowerCommandsforVisualStudio
Go to `Tools > Options > Productivity Power Tools > PowerCommands`
Enable `Format documents on save` and `Remove and Sort Usings on save`